docs/data-sources/volume.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_volume Data Source
+
+Retrieves details about [databricks_volume](../resources/volume.md) that was created by Terraform or manually. 
+A volume can be identified by its three-level (fully qualified) name (in the form of: `catalog_name`.`schema_name`.`volume_name`) as input. This can be retrieved programmatically using [databricks_volumes](../data-sources/volumes.md) data source.
+
+## Example Usage
+
+* Retrieve details of all volumes in in a _things_ [databricks_schema](../resources/schema.md) of a  _sandbox_ [databricks_catalog](../resources/catalog.md):
+
+```hcl
+data "databricks_volumes" "all" {
+  catalog_name = "sandbox"
+  schema_name  = "things"
+}
+
+data "databricks_volume" {
+  for_each = data.datatbricks_volumes.all.ids
+  name     = each.value
+}
+```
+
+* Search for a specific volume by its fully qualified name
+
+```hcl
+data "databricks_volume" "this" {
+  name = "catalog.schema.volume"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) a fully qualified name of [databricks_volume](../resources/volume.md): *`catalog`.`schema`.`volume`*
+
+
+## Attribute Reference
+
+This data source exports the following attributes:
+
+* `volume_info` - TableInfo object for a Unity Catalog table. This contains the following attributes:
```

Comment:
We need to mention `id` because we set it...

Reply:
added to docs in 714d0c2432d14fdfcf765f9d5204db09936f29c1